merging_health.gms

```diff
@@ -0,0 +1,111 @@
+********************************************************************************
+$ontext
+
+   CAPRI project
+
+   GAMS file : merging_health.gms
+
+   @purpose  :  To match consumption scenarios from Springmann et al. (2018) to CAPRI's human intake outputs
+
+      See Springmann, M., Wiebe, K., Mason-D'Croz, D., Sulser, T. B., Rayner, M., & Scarborough, P. (2018).
+      Health and nutritional aspects of sustainable diet strategies and their association with environmental impacts: a global modelling analysis with country-level detail
+      The Lancet Planetary Health Volume 2, Issue 10, October 2018, Pages e451-e461
+      https://www.sciencedirect.com/science/article/pii/S2542519618302067
+
+   @author   :
+   @date     :
+   @since    :
+   @refDoc   :
+   @seeAlso  :
+   @calledBy : cons_0203.gms (note: Springmann et al. (2018) sets will be called by data_prep.gms (?))
+
+$offtext
+********************************************************************************
+
+* --- auxiliary sets:
+
+set diet_scn_p diet scenarios /"BMK", "FLX", "PSC", "VEG", "VGN" /;
+set unit_wp consumption_unit(net of waste)  /"g/d_w"/;
+
+
+*------------------------------------------------------------------------------
+*   COLLECT HUMAN INTAKE REFERENCE FROM CAPRI
+*------------------------------------------------------------------------------
+
+parameter p_nutrientCont(rows,usda_cont)           "nutrient content per kg (mainly from USDA)";
+parameter p_NCNC_CONT(rows,ncnc)                   "nutrient content per kg mapped from USDA texts to CAPRI codes";
+
+$GDXIN %datdir%\arm\nutrient_cont_usda.gdx
+$LOAD  p_nutrientCont
+$GDXIN
+
+p_NCNC_CONT(rows,NCNC) = sum(ncnc_to_usda(ncnc,usda_cont),p_nutrientCont(rows,usda_cont));
+
+* --- files converting regions/countries and food groups from CAPRI into EAT-Lancet categorization (from A.LEIP):
+
+$include '%curdir%\pol_input\epnf\tasteshift\mapIMPACTregions.gms'
+$include '%curdir%\pol_input\epnf\tasteshift\EATmap2capri.gms'
+
+* --- files from Springmann et al:
```

Comment:
It would be good if you add all files needed, also data_0203 and parameters .. and the others used below. 
My suggestion is that you create a folder under gams called 'diet' where they can all go (later we can fine tune which ones should rather go under 'dat' etc.)